src/AutoRest.CSharp.V3/Generation/Writers/ClientWriter.cs

```diff
@@ -58,6 +67,56 @@ public void WriteClient(CodeWriter writer, Client client)
             }
         }
 
+        private void WriteManagementClientCtors(CodeWriter writer, Client client, Configuration configuration)
+        {
```

Comment:
This is pretty hardcoded but should work for the first iteration, will improve as we gather more feedback.